Modules/arraymodule.c

```diff
@@ -1115,18 +1115,42 @@ array_array_count(arrayobject *self, PyObject *v)
 array.array.index
 
     v: object
+    start: Py_ssize_t = 0
+    stop: Py_ssize_t = 9223372036854775807
```

Comment:
What does this 9223372036854775807 mean? I have idea, but this looks totally confusing.

And there is an integer overflow on 32-bit platform.

Reply:
I found it somewhere (in cpython), though I can't remember where. I'll try to readjust with clinic, taking list_index() as an example, and commit the updates asap.